WordPress/twentytwentyfive
# Try: Escape apostrophe in translation functions.

**Description**


Replacing `&rsquo;` for an escaped apostrophe (`\'`) in translatable texts.

**Screenshots**

- N/A

**Testing Instructions**

1. Create a page.
2. Insert all the patterns modified in this PR.
3. Confirm the texts are looking as they were before.
